core/src/main/scala/org/apache/spark/status/api/v1/OneApplicationResource.scala

```diff
@@ -52,6 +53,25 @@ private[v1] class AbstractApplicationResource extends BaseAppResource {
   @Path("executors")
   def executorList(): Seq[ExecutorSummary] = withUI(_.store.executorList(true))
 
+  @GET
+  @Path("executorSummary")
```

Comment:
should be executorMetricsSummary or perhaps this should be executormetricsDistribution or Quantiles or something.  The metrics themselves are in the /executors path in ExecutorSummary, correct?